### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by JamesWiresmith** [aldanor/hdf5-rust#299] [General]
Hi All,

I wanted to raise again concerns about the status of this project. It seems that many issues are blocked on @aldanor. Right now I've been able to move forward with a fork but there are many key issues raised with the age of the last update (3 years old on crates.io).

@aldanor I totally appreciate that interests move on in open source but can we find a way to unstick this project so we can bring it up to date with HDF5. I'm happy to offer what assistance I can with my limited experience of the project.

**Comment by magnusuMET** [comment]
A fork has been created which has new releases on crates.io: https://github.com/aldanor/hdf5-rust/issues/295

**Comment by aldanor** [comment]
Hi all - apologies for disappearing for a long while, I haven't had much time even to sleep lately :)

@magnusuMET thanks for keeping the project alive; I have a few suggestions though:
- To avoid confusion on crates.io (with two different forks and two different suffixes of crates), how about I try to transfer the ownership of `hdf5-*` crates to yourself? I haven't done this before and looks like I will have to resolve this via email by contacting crates.io management but I can give it a try. It may cause a temporary confusion due to switching to original names but it has already happened a few times (even at the very start when it was hdf5-rs because hdf5 name was taken and was later transferred to me).
- How about we start a separate org where @magnusuMET would have admin rights? This has multiple benefits - e.g. we can keep all the discussions and issues and PRs alive in the main live repository. It would be sad to lose them all along the way.
  - Alternatively, I can transfer this repository itself under @magnusuMET's private ownership but the org seems better?

**Reply by magnusuMET** [reply]
Thank you @aldanor for taking the time to respond here!

I would like to see the original crate names kept and deprecate the `-metno` suffixed versions. It should be possible to add any owner on crates.io directly, see e.g. https://crates.io/crates/hdf5/settings (and similarly for all the other hdf5-* crates) without having to go through the management.

When we transferred the `netCDF` crate we were allowed to use the [georust organisation](https://github.com/georust). I can certainly see the `hdf5-rust` crate as a valuable asset to the geospatial community.

**Reply by aldanor** [reply]
@magnusuMET I just realized I dropped out again having forgot about this due to being extremely busy in real life so to speak 🤦 In order to maintain issues and history etc, I can immediately transfer ownership either to yourself or georust org if you can arrange that, and you can then decide if you want to retain the crate name - you have been added as an owner to `hdf5` crate on crates.io and should have received an invite. Please lmk if there's anything else I can do.

**Reply by magnusuMET** [reply]
I think the best steps forward would be to move the repository to me, and I'll forward it to an organisation or create one if necessary. If you could add me as an owner for the following crates that would be great!

https://crates.io/crates/hdf5-derive
https://crates.io/crates/hdf5-src
https://crates.io/crates/hdf5-sys
https://crates.io/crates/hdf5-types

**Reply by aldanor** [reply]
@magnusuMET 👍  Ok I'll try to do that tomorrow and will keep you posted!
